packages/http-server/src/types.ts

```diff
@@ -4,6 +4,7 @@ import { FastifyInstance } from 'fastify';
 export interface IPrismHttpServerOpts {
   components?: PickRequired<TPrismHttpComponents, 'logger'>;
   config: IHttpConfig;
+  cors: boolean;
```

Comment:
In theory couldn't dynamic mocking be a server config as well?

Reply:
And validation for that matter?

Reply:
Not exactly — Prism has basically the `Http` and the `HttpServer` packages

The Http package is able to run the mocking and forwarding functionalities without having to require any server. It's basically this:

```ts

const prism = createPrism(operations, config); // config is IHttpConfig
prism.process(inputRequest) // get an answer
```

For this, it makes sense to know whether you want the input validation (which is in the `inputRequest`) or the output validation (which could be come from the mocker or the forwarded response from the upstream server).

In this case, `cors` does not make any sense; it should be a configuration parameter of the `http-server` — which is used by Studio and the CLI package.

I'm merging this, but in meantime we can keep the conversation going here.

Reply:
makes sense